cmake/kokkos_arch.cmake

```diff
@@ -33,7 +33,7 @@ KOKKOS_DEPRECATED_LIST(ARCH ARCH)
 
 SET(HOST_ARCH_ALREADY_SPECIFIED "")
 MACRO(DECLARE_AND_CHECK_HOST_ARCH ARCH LABEL)
-  KOKKOS_ARCH_OPTION(${ARCH} HOST "${LABEL}")
+  KOKKOS_ARCH_OPTION(${ARCH} HOST "${LABEL}" TRUE)
```

Comment:
I don't get the trailing argument.

Reply:
The trailing argument describes the dependency that needs to be satisfied for the variables to be shown. Since the host architectures work with all host backends, they are always there.

Reply:
OK I guess I was confused by TRUE instead of ON